src/main/java/com/example/demo/models/SignupModel.java

```diff
@@ -0,0 +1,8 @@
+package com.example.demo.models;
+
+public class SignupModel {
+	public String firstName;
+	public String lastName;
+	public String email;
+	public String password;
```

Comment:
let's respect encapsulation principle and make all fields private and access them with getters and setters

if you find repetitive generating getters and setters, you can use Lombok library to handle this with anotations

this is issue in whole PR, public and protected should be used only when really necessary, everything else should be private